#!/usr/bin/env python

# Must be an executable file. Check the permissions!

PACKAGE = "dvs_integrator"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# Add a parameter (type double) using  slider in [0, 20] and default value = 5
gen.add("Cutoff_frequency", double_t, 0,
"Cutoff frequency (alpha) rad/s", 5.0, 0, 20)

exit(gen.generate(PACKAGE, "integrator_node", "dvs_integrator"))
